website/.spelling

```diff
@@ -1328,6 +1328,7 @@ valueFormat
 IOConfig
 compactionTask
 compactionTasks
+storeCompactionState
```

Comment:
This will probably not be needed if we quote the field name in the docs as required.